frontend/src/utils/app.ts

```diff
@@ -17,3 +17,16 @@ export const isCloudUser = (): boolean => {
 
 	return hostname?.endsWith('signoz.cloud');
 };
+
+export const checkVersionState = (
```

Comment:
```suggestion
export const checkVersionState = (
export const checkVersionState = (currentVersion: string, latestVersion: string): boolean => {
  const versionCore = currentVersion?.split('-')[0];
  return versionCore === latestVersion;
};

```

We can remove the unnecessary if statement as `split` function will return an array containing the original string as its single element if it doesn't find the specified separator in the string